publication-model/src/main/java/no/unit/nva/model/associatedartifacts/file/File.java

```diff
@@ -60,8 +58,8 @@ public abstract class File implements JsonSerializable, AssociatedArtifact {
             "https://creativecommons.org/publicdomain/zero/1.0"), "RightsReserved",
                                                               URI.create("http://rightsstatements.org/vocab/InC/1.0/"));
 
-    public static final String MISSING_LICENSE = "The file is not annotated as an administrative agreement and should" +
-                                                 " have a license";
+    public static final String MISSING_LICENSE =
+        "The file is not annotated as an administrative agreement and should" + " have a license";
```

Comment:
"This file is served to the public and should therefore have a license", I suspect.

Reply:
Or "is open"